test/integration/targets/azure_rm_sqlserver/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/azure
+destructive
```

Comment:
You will need to add `posix/ci/cloud/group2/azure` to this list so the tests actually run in CI